src/js/files/components/UploadTime.tsx

```diff
@@ -0,0 +1,43 @@
+import { formatDuration, intervalToDuration } from "date-fns";
+import React from "react";
+import styled from "styled-components";
+import { getFontSize, getFontWeight } from "../../app/theme";
+
+const StyledUploadInformation = styled.div`
+    display: flex;
+    justify-content: space-between;
+    font-size: ${getFontSize("md")};
+    font-weight: ${getFontWeight("normal")};
+    margin-top: 5px;
+`;
+
+function isDurationZero(duration: Duration): boolean {
+    return Object.values(duration).every(value => value === 0 || value === undefined);
+}
+
+/**
+ * Displays remaining time and upload speed.
+ * @prop remaining - remaining time in seconds
+ * @prop uploadSpeed - upload speed in bytes per second
+ */
+export function UploadTime({ remaining, uploadSpeed }: { remaining: number; uploadSpeed: number }): JSX.Element {
```

Comment:
I would define the type `UploadTimeProps` with the typings and docstrings to keep it consistent with the codebase